File: Section10_Probability-for-Robotics/bumperbot_ws/src/bumperbot_controller/nodes/simple_controller_node.cpp
#include <ros/ros.h>
#include "bumperbot_controller/simple_controller.h"


int main(int argc, char **argv)
{
    ros::init(argc, argv, "simple_controller");
    ros::NodeHandle nh;
    ros::NodeHandle pnh("~");
    double wheel_radius, wheel_separation;
    pnh.getParam("wheel_radius", wheel_radius);
    pnh.getParam("wheel_separation", wheel_separation);
    SimpleController controller(nh, wheel_radius, wheel_separation);
    ros::spin();

    return 0;
}

File: Section11_Sensor-Fusion/bumperbot_ws/src/bumperbot_controller/nodes/simple_controller_node.py
#!/usr/bin/env python3

import rospy
from bumperbot_controller.simple_controller import SimpleController


if __name__== '__main__':
    rospy.init_node('simple_controller')
    wheel_radius = rospy.get_param('~wheel_radius')
    wheel_separation = rospy.get_param('~wheel_separation')
    controller = SimpleController(wheel_radius, wheel_separation)

    rospy.spin()